src/VuePictureSwipe.vue

```diff
@@ -3,6 +3,7 @@
     <div class="my-gallery" itemscope itemtype="http://schema.org/ImageGallery">
 
       <figure
+          v-show="index > 0 && !singleThumb"
```

Comment:
Maybe we could try to be more explicit (by replacing `singleThumb` by `singleThumbnail`:

```suggestion
          v-show="index > 0 && !singleThumbnail"
```